Interface/AddOns/oUF_NeavRaid/tags.lua

```diff
@@ -45,5 +46,6 @@ end
 oUF.Tags.Events['name:raid'] = 'UNIT_NAME_UPDATE'
 oUF.Tags.Methods['name:raid'] = function(unit)
     local name = UnitName(unit)
+    if (name == nil) then return '' end
```

Comment:
Seems like a Blizzard bug or something, does this really happen that `name` is `nil`?